src/core/lib/event_engine/windows/socket.h

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 gRPC authors.
```

Comment:
suggest renaming to win_socket.{h,cc} to match the class name

Reply:
Done